stumpy/gpu_aamp.py

```diff
@@ -9,14 +9,15 @@
 import numpy as np
 from numba import cuda
 
-from . import core, config
+from . import core, config, gpu_stump
 
 logger = logging.getLogger(__name__)
 
 
 @cuda.jit(
     "(i8, f8[:], f8[:], i8, f8, f8[:], f8[:], f8[:], b1[:], b1[:],"
-    "i8, b1, i8, f8[:, :], i8[:, :], b1)"
+    "i8, b1, i8, f8[:, :], f8[:], f8[:], i8[:, :], i8[:], i8[:], b1,"
+    "i8[:], i8, i2)"
```

Comment:
here, we used `i2` (not `i8`) as signature of `k`.

Reply:
`i8` please (see comment above)